apps/server/src/utils/fileManagement.ts

```diff
@@ -114,3 +114,11 @@ export async function dockerSafeRename(oldPath: PathLike, newPath: PathLike) {
   await copyFile(oldPath, newPath);
   await unlink(oldPath);
 }
+
+/**
+ * finds potential file index number in our (*) format
+ * @param baseName the base name of the project file, without extention af path
```

Comment:
question: does it matter if the path and extension are there? if so, would we be better off writing some simple logic to extract the number in parenthesis instead of regex?

```suggestion
 * @param baseName the base name of the project file, without extension or path
```

Reply:
yes this only works without extension, it could be changed to work with it but
all that is already handled in the `generateUniqueFileName` where the information about the path and extension is needed anyway

Reply:
I would suggest simplifying the regex to a simple function that looks for the last parenthesis. That should mean that it doesn’t care about extensions